package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "thankful",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "Thankful",
   "main": "index.js",
   "scripts": {
     "test": "jest --coverage",
-    "lint": "eslint --ignore-path=.gitignore --ext .vue .",
+    "lint": "eslint --ignore-path=.gitignore --ext .vue --ext .js .",
```

Comment:
Did you check that it still runs for .Vue files like this?

Reply:
Yes, see the docs for eslint